### PR TITLE
[Edit] HTML Add Media: Links #3231

### PR DESCRIPTION
**Pull request by maxx-l**
### Description

Added a section about the title attribute in links.md 

### Type of Change

- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/links/links.md [hunk 1]
```diff
@@ -85,3 +85,13 @@ The two primary attributes are:
...
+### Links attribute
+
+The `title` attribute can be optionally defined to provide extra, informative details about a linked website. This feature aids in enhancing or elaborating on the intended function of a link, which the recipient should be aware of before clicking on it.
+
+```html
+<!-- This link will help users to predict what will happen if they click on it. -->
+
+<a href="https://www.codecademy.com/resources/docs" title="Go to codecademy documentation">Visit our Tutorial</a>
+```
+
```
Comment (SSwiniarski) [inline content/html/concepts/links/links.md:97]: Can we have an image of a cursor hovering over the link with the pop-up text?  The issue specifically wanted images.

File: content/html/concepts/links/links.md [hunk 2]
```diff
@@ -85,3 +85,13 @@ The two primary attributes are:
...
 </html>
 ```
+### Links attribute
+
+The `title` attribute can be optionally defined to provide extra, informative details about a linked website. This feature aids in enhancing or elaborating on the intended function of a link, which the recipient should be aware of before clicking on it.
+
+```html
+<!-- This link will help users to predict what will happen if they click on it. -->
+
+<a href="https://www.codecademy.com/resources/docs" title="Go to codecademy documentation">Visit our Tutorial</a>
```
Comment (SSwiniarski) [inline content/html/concepts/links/links.md:95]:
```suggestion
<a href="https://www.codecademy.com/resources/docs" title="Go to Codecademy Docs">Learn about code.</a>
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@maxx-l I did a first review and left two comments.

**Comment by maxx-l** [comment]
Hi @SSwiniarski ,

Thanks for the review. I added your first comment to this commit.
I am trying to add a gif but I am struggling to display it on my side.  
I used this markdown  found [here ](https://www.codecademy.com/resources/docs/contribution-guide/writing-your-doc)
`Example`

any suggestion?
Cheers

**Comment by caupolicandiaz** [comment]
@maxx-l There hasn't been any any activity on this PR, are you still available to complete this?

**Pull request closed without merging**